packages/accordions/stories/2-Accordion.stories.tsx

```diff
@@ -0,0 +1,320 @@
+      summer purslane earthnut pea tomato spring onion azuki bean gourd. Gumbo kakadu plum komatsuna
+      black-eyed pea green bean zucchini gourd winter purslane silver beet rock melon radish
+      asparagus spinach.`
+    },
+    {
+      id: 'section-4',
+      header: 'Soko radicchio bunya',
+      headerDetail: 'nuts gram dulse silver beet parsnip napa cabbage lotus root',
+      panelContent: `Soko radicchio bunya nuts gram dulse silver beet parsnip napa cabbage lotus root
+      sea lettuce brussels sprout cabbage. Catsear cauliflower garbanzo yarrow salsify chicory garlic
+      bell pepper napa cabbage lettuce tomato kale arugula melon sierra leone bologi rutabaga
+      tigernut. Sea lettuce gumbo grape kale kombu cauliflower salsify kohlrabi okra sea lettuce
+      broccoli celery lotus root carrot winter purslane turnip greens garlic. Jícama garlic courgette
+      coriander radicchio plantain scallion cauliflower fava bean desert raisin spring onion chicory
+      bunya nuts. Sea lettuce water spinach gram fava bean leek dandelion silver beet eggplant bush.`
+    }
+  ];
+
+  const [expandedSections, setExpandedSections] = useState([0]);
+  const isExpandAll = expandedSections.length === 5;
+  const isCollapseAll = expandedSections.length === 0;
+
+  const expandAll = useCallback(() => setExpandedSections([0, 1, 2, 3, 4]), []);
+
+  const collapseAll = useCallback(() => setExpandedSections([]), []);
+
+  const onChange = (index: number) => {
+    setExpandedSections(sections =>
+      expandedSections.includes(index)
+        ? sections.filter(section => section !== index)
+        : [...expandedSections, index]
+    );
+  };
+
+  return (
+    <>
+      <StyledButtonWrapper>
+        <Button onClick={expandAll} disabled={isExpandAll}>
+          Expand all
+        </Button>
+        <Button onClick={collapseAll} disabled={isCollapseAll}>
+          Collapse all
+        </Button>
+      </StyledButtonWrapper>
+      <Accordion
+        level={3}
+        isBare={isBare}
+        onChange={onChange}
+        isCompact={isCompact}
+        expandedSections={expandedSections}
+      >
+        {accordionSections.map(section => (
+          <Accordion.Section key={section.id}>
+            <Accordion.Header>
+              <Accordion.Label>
+                {section.header}
+                <StyledSM forwardedAs="span">{section.headerDetail}</StyledSM>
+              </Accordion.Label>
+              <ButtonGroup isCompact={isCompact} />
+            </Accordion.Header>
+            <Accordion.Panel>{section.panelContent}</Accordion.Panel>
+          </Accordion.Section>
+        ))}
+      </Accordion>
+    </>
+  );
+};
+
+Advanced.args = {
+  isCompact: false,
+  isBare: false
+};
+
+Advanced.parameters = {
+  docs: {
+    description: {
+      story: `
+  The \`Accordion\` component is implemented using the [W3C Accordion pattern](https://www.w3.org/TR/wai-aria-practices/#accordion).
```

Comment:
This repeat from the section above might not be necessary.

Reply:
Removed.